Tobysmith.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Toby Smith</title>
+    <link rel="stylesheet" href="style5.css" />
+    <script src="https://kit.fontawesome.com/ca44700445.js" crossorigin="anonymous"></script>
+    <script src="script.js"></script>
+</head>
+<body>
+
+    <div id="page-container">
```

Comment:
Semantic HTML has been used well in your index.html, ensure to use in all your HTML pages even if there is not a lot of content. Semantic HTML is great as it makes it easy for us as developers and the machine to read code, but also makes code more accessible!